modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -156,7 +156,7 @@ private Boolean getGeneratorPropertyDefaultSwitch(final String key, final Boolea
         return defaultValue;
     }
 
-    private void configureGeneratorProperties() {
+    protected void configureGeneratorProperties() {
```

Comment:
This should not be marked as protected as it reduces maintainability. If necessary for tests, please remove `private` to make it package private.

Reply:
This was changed to package private

Reply:
This was changed to package private